components/scream/src/physics/shoc/shoc_diag_second_moments_srf_impl.hpp

```diff
@@ -0,0 +1,44 @@
+
+#ifndef SHOC_DIAG_SECOND_MOMENTS_SRF_IMPL_HPP
+#define SHOC_DIAG_SECOND_MOMENTS_SRF_IMPL_HPP
+
+#include "shoc_functions.hpp" // for ETI only but harmless for GPU
+#include "physics_functions.hpp" // also for ETI not on GPUs
+
+namespace scream {
+namespace shoc {
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>
+::shoc_diag_second_moments_srf(
+    const Int& shcol, 
```

Comment:
schol is no longer needed in the C++ impl of this function

Reply:
I was trying to keep the same argument as in Fortran code. If you want to remove it, we could do.

Reply:
Yes, lets remove it. We are already differing from fortran in the other arguments.

Reply:
sounds good, done!